pitch.md

```diff
@@ -23,38 +89,146 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
-
+I think this is a pretty "big" idea.  I think that if I am successfully able to
+implement this DSL, it really can provide an alternative way to take notes on
+the computer for more quantitative subjects.  While laptops and students taking
+notes with laptops are ubiquitous in schools, I don't think we can say that 
+society has found an effective way to improve the classroom experience in lecture
+based classes through computers other than greater access to powerpoint slides.
+I think the student experience could be greatly improved by this DSL. 
+
+If I am able to navigate the environment that the DSL will run in, I don't think
+it is overly ambitious to complete this project.  If working with the notebook
+API to parse code and create visualizations is straightforward, I don't see a
+reason why I couldn't complete this project.  That being said, requiring an
+outside application and therefore outside API as a major part of my project puts
+me out of control for a big chunk of my project. I won't really know how that 
+process will be, so finding an application that allows me to focus on language
+design rather than "systems" programming will be crucial and determine how
+ambitious this project really is. 
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+This is a good project because it solves a real problem, and one that should be
+solved by a language rather than an app.  Hopefully if this project is 
+successful, it can provide an example of a clear use case where a DSL is superior 
+to an app.  
+
+A major challenge I expect to face is finding the best note-taking app to have
+my DSL run in.  Given that I need an outside app to make my DSL work, I expect
+that I'll run into some obstacles in implementing my DSL in the context of
+another app that I don't get to control.  I have already started my research
+on this challenge, and starting early in figuring this out is the best plan of
+action to overcome this. 
+
+Another challenge will be figuring out how best to design the language.  I have
+a strong bias towards a syntax I'd came up with in the `project-ideas` assignment,
+which would not provide a lot of opportunity for design aspects of the project.
+To solve this, I just need to put a lot more thought and do more surveying to see
+what actually is the preferred design among potential users rather than just 
+jumping to my own conclusions. 
+
+I think if I am spending too much time wrestling with the API of the 
+external note-taking application, I would be off-track.  This would make it more
+of a systems project than a design project, so I would have to reconsider my 
+idea, or think of an alternative environment to implement it for. 
```

Comment:
If this does happen, you could maybe at some point make the decision to stop focusing on the implementation and just come up with a solid specification for the language. At least that way you'd get to go through the whole design process.